### PR TITLE
Toggle switch new design and new icons + adding vite svg plugin

### PR DESCRIPTION
**Pull request by GiliStern**
https://github.com/talyaron/delib-5-p/assets/25402124/6778c3b7-1165-47ee-ba57-4da88e9a5a45

### CODE REVIEW

File: src/view/components/switch/CustomSwitch.module.scss [hunk 1]
```diff
@@ -1,37 +1,60 @@
...
+            color: white;
+            left: 150px;
+        }
+
+        .label {
+            background-color: #74adf0;
+            color: white;
+        }
+    }
+
```
Comment (vladbyk89) [inline src/view/components/switch/CustomSwitch.module.scss:26]: Could we make the size of the switch responsive? It looks out of place on a mobile phone.

Comment (GiliStern) [reply src/view/components/switch/CustomSwitch.module.scss:26]: Looks better this way?
image

Comment (GiliStern) [reply src/view/components/switch/CustomSwitch.module.scss]: I didn't make it responsive to the level of changing size when window size changes, but I fixed my negative margins and now it's aligned better

Comment (vladbyk89) [reply src/view/components/switch/CustomSwitch.module.scss]: I do think we should make it smaller if the screen size becomes very small, but I will let @talyaron make that call :)

**Review by talyaron — APPROVED** [review]

**Review by talyaron — APPROVED** [review]
Well done

**Pull request merged**